src/components/atoms/Flex/index.tsx

```diff
@@ -33,21 +35,28 @@ const Flex: React.FC<Props> = ({
   shrink,
   flex,
   gap,
+  role,
+  hidden,
 }) => {
-  const styles = {
-    display: "flex",
-    flexDirection: direction,
-    alignItems: align,
-    justifyContent: justify,
-    flexWrap: wrap,
-    flexBasis: basis,
-    flexGrow: grow,
-    flexShrink: shrink,
-    flex: flex,
-    gap: gap, // TODO: Safari doesn't support this property and please develop polyfill
-  };
   return (
-    <div className={className} style={styles} onClick={onClick} data-testid={testId}>
+    <div
+      className={className}
+      role={role}
+      style={{
+        display: "flex",
+        flexDirection: direction,
+        alignItems: align,
+        justifyContent: justify,
+        flexWrap: wrap,
+        flexBasis: basis,
+        flexGrow: grow,
+        flexShrink: shrink,
+        flex,
+        gap, // TODO: Safari doesn't support this property and please develop polyfill
+      }}
+      onClick={onClick}
+      data-testid={testId}
+      aria-hidden={!hidden}>
```

Comment:
Why is it negated? Shouldn't `aria-hidden` = true if `hidden` is true? 
Instead wouldn't it make more sense to make `hidden`'s default true. That way, devs would have to specifically set the Flex to false so that screen readers know it is interactive. Unless the other way makes more sense and unless stated ALL Flex components are considered interactive.

Reply:
Fixed: [`5359564` (#284)](https://github.com/reearth/reearth-web/pull/284/commits/5359564bad4035b2cd6c8805106262c713484aa4)